### PR TITLE
create daily notification

### PR DESCRIPTION
**Pull request by amitkle**

### CODE REVIEW

File: app/src/main/java/com/example/coronadiagnosticapp/ui/fragments/welcome/AutostartUtils.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import com.example.coronadiagnosticapp.BuildConfig;
+import com.example.coronadiagnosticapp.R;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class AutostartUtils {
+
+    public static void requestAutostartPermissions(Context context) {
+        final List<Intent> SETTINGS_INTENTS = Arrays.asList(
```
Comment (svechinsky) [inline app/src/main/java/com/example/coronadiagnosticapp/ui/fragments/welcome/AutostartUtils.java:27]: Just confirming, this list is everything we need? Cause some providers are missing here

File: app/src/main/java/com/example/coronadiagnosticapp/ui/activities/Reminder/AlarmNotificationReceiver.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                .setContentTitle(context.getString(R.string.reminder_notification_title))
+                .setContentIntent(pendingIntent) // Open the MainActivity after the user pressed the notification.
+                .setContentText(context.getString(R.string.reminder_notification_content));
+
+        NotificationManager notificationManager = (NotificationManager)context.getSystemService(Context.NOTIFICATION_SERVICE);
+
+        if (android.os.Build.VERSION. SDK_INT >= android.os.Build.VERSION_CODES.O) {
+            // Only from version O we need to use NotificationChannel.
+            // This allows features like "block all notification like that"
+            int importance = NotificationManager. IMPORTANCE_HIGH ;
```
Comment (ofirassif) [inline app/src/main/java/com/example/coronadiagnosticapp/ui/activities/Reminder/AlarmNotificationReceiver.java]: Is there an extra space here?

File: app/src/main/java/com/example/coronadiagnosticapp/ui/activities/Reminder/RegisterNotificationService.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    public IBinder onBind(Intent intent) {
+        return null;
+    }
+
+    @Override
+    public void onCreate() {
+        Log.d(TAG, "onCreate RegisterNotificationService.");
+
+        super.onCreate();
+        setDailyNotification(10, 0);
```
Comment (ofirassif) [inline app/src/main/java/com/example/coronadiagnosticapp/ui/activities/Reminder/RegisterNotificationService.java]: Const it out?

File: app/src/main/java/com/example/coronadiagnosticapp/ui/fragments/welcome/AutostartUtils.java (hunk 1 above)
Comment (amitkle) [reply app/src/main/java/com/example/coronadiagnosticapp/ui/fragments/welcome/AutostartUtils.java:27]: Talked about it on whatsapp.


Those are the only providers I read that needs this permission.
Though I didn't test this on other devices than my - xiaomi, and pixal emolator. The pixal indeed didn't need any other permission.

File: app/src/main/java/com/example/coronadiagnosticapp/ui/activities/Reminder/AlarmNotificationReceiver.java (hunk 2 above)
Comment (amitkle) [reply app/src/main/java/com/example/coronadiagnosticapp/ui/activities/Reminder/AlarmNotificationReceiver.java]: oh yes, lol
will fix it

File: app/src/main/java/com/example/coronadiagnosticapp/ui/activities/Reminder/RegisterNotificationService.java (hunk 3 above)
Comment (amitkle) [reply app/src/main/java/com/example/coronadiagnosticapp/ui/activities/Reminder/RegisterNotificationService.java]: done

**Comment by svechinsky** [comment]
Yalla looks good, I'll merge after you fix the last conflicts

**Pull request merged**